src/input_common/udp/client.h

```diff
@@ -122,14 +125,25 @@ class Client {
     void StartCommunication(std::size_t client, const std::string& host, u16 port,
                             std::size_t pad_index, u32 client_id);
     void UpdateYuzuSettings(std::size_t client, const Common::Vec3<float>& acc,
-                            const Common::Vec3<float>& gyro, bool touch);
+                            const Common::Vec3<float>& gyro);
+
+    // Returns an unused finger id, if there is no fingers avaliable udp_finger_id size will be
+    // returned
+    size_t GetUnusedFingerID() const;
```

Comment:
```suggestion
    std::optional<size_t> GetUnusedFingerID() const;
```

And return std::nullopt in the case none are available.